activerecord/lib/active_record/type/adapter_specific_registry.rb

```diff
@@ -82,7 +82,7 @@ def priority_except_adapter
 
       private
 
-      def matches_adapter?(adapter: nil, **)
+      def matches_adapter?(adapter: nil, **kwargs)
```

Comment:
Hm I'm not sure, `**` is perfectly valid Ruby, pretty much like `*`, isn't it? If YARD is triggering an error, that's unlikely to be this implementation's fault? Maybe YARD should be fixed instead?